frontend/src/routes/PostCard.svelte

```diff
@@ -0,0 +1,79 @@
+<script lang="ts">
+	export let post: {
```

Comment:
nit: if we can declare type elsewhere, I can reuse the type in in `lib` folder :)